src/main/java/com/ita/if103java/ims/config/SecurityConfig.java

```diff
@@ -21,13 +41,36 @@ protected void configure(HttpSecurity http) throws Exception {
             .sessionCreationPolicy(SessionCreationPolicy.STATELESS)
             .and()
             .authorizeRequests()
-            .antMatchers("/**").permitAll()
-            .anyRequest().authenticated();
+            .antMatchers("/",
+                "/signin",
+                "/forgot-password",
+                "/forgot-password/reset-password",
+                "/signup").permitAll()
+            .anyRequest().authenticated()
+            .and()
+            .apply(new JwtTokenFilterConfigurer(jwtTokenProvider));
     }
 
     @Override
     public void configure(WebSecurity web) throws Exception {
-        web.ignoring().antMatchers("/v2/api-docs", "/configuration/ui", "/swagger-resources", "/configuration/security", "/swagger-ui.html", "/webjars/**");
+        web.ignoring()
+            .antMatchers("/v2/api-docs",
+                "/configuration/ui",
+                "/swagger-resources",
+                "/configuration/security",
+                "/swagger-ui.html",
+                "/webjars/**");
```

Comment:
I think we can combine these two lists of exclusion into one.
Could you please try to move "swagger" exclusion into "signin" exclusion list?

Reply:
Of course)